docs/quickstart/google-cloud-platform.md

```diff
@@ -0,0 +1,114 @@
+# Google Cloud Platform
+
+## Requirements
+
+This quick start assumes you have administrator access to your Google Cloud Platform instance. 
+
+## Setting Up your VM  
```

Comment:
I commented something similar on the AWS and Azure quickstarts but it was lost to the noise.

We should explain to people that we're first creating the control plane instance (using friendlier language). It is not clear from these tutorials that we're creating the workspace manager VM as opposed to the workspace VM.